### ISSUE TITLE
Resize method is not in the Flicking instance in vue-flicking

### ISSUE THREAD

**Issue by kazvaggos** [naver/egjs-flicking#458]
## Description
Hello. According [docs](https://naver.github.io/egjs-flicking/docs/api/Flicking#resize), I should be able to call resize method for the vue-instance of flicking. However, when logging the instance, the method does not exist. It would nice to have it in the instance, since `autoResize` option adds resize event listener in the window for every carousel rendered in the view. With too many carousels, it hits the performance of the page with all these listeners.
For now I implemented it myself by looking at the code base
```js
function resize() {
  const {flicking} = this.$refs;
  const {activePanel, camera, control, renderer, viewport} = flicking;
  const prevProgressInPanel = activePanel ? flicking.getProgressInPanel(activePanel) : 0;

  viewport.resize();
  renderer.forceRenderAllPanels().then(() => {
    renderer.updatePanelSize();
    renderer.render().then(() => {
      camera.updateAlignPos();
      camera.updateRange();
      camera.updateAnchors();
      control.updatePosition(prevProgressInPanel).then(() => {
        flicking.updateInput();
      });
    });
  });
}
```
However, is it possible to add it in the instance as docs indicate?

## Steps to check or reproduce
1. Mount carousel and add a attribute ref="flicking" in flicking component
2. If you run `this.$refs.flicking.resize()` you will run to an error like `resize is not a function`
3. If you try `console.log(this.$refs.flicking)` you can't find anywhere the `resize` method

**Comment by WoodNeck** [comment]
Hello @kazvaggos!
Thanks for reporting the issue, I've just created the PR that fixes this issue :)
I'll leave a comment here after releasing that fix

**Comment by WoodNeck** [comment]
We've released a new version `v4.1.0` that fixes this issue, please test that instead!
